<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1165</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="group_redis">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>171</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="title">
     <string>REDIS</string>
    </property>
    <widget class="QLabel" name="version_of_redis_title">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Version of REDIS :</string>
     </property>
    </widget>
    <widget class="QLabel" name="version_of_redis_label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="status_redis_title">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>40</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
    <widget class="QLabel" name="status_redis">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>40</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="start_redis_button">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>130</y>
       <width>112</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stop_redis_button">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>130</y>
       <width>112</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_vagrant">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>561</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Vagrant</string>
    </property>
    <widget class="QLabel" name="vagrant_boxes_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Vagrant machine:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_vagrant_boxes">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>191</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Path to Vagrant machines:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="open_browse_files_vagrant">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>71</y>
       <width>112</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="agent_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>1141</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Agent</string>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>361</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="install_redis_button">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>180</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Install Redis</string>
    </property>
   </widget>
   <widget class="QPushButton" name="install_vagrant_button">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>180</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Install Vagrant</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>install_redis_button</tabstop>
  <tabstop>install_vagrant_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
